requirements.txt

```diff
@@ -1,4 +1,4 @@
-google-cloud-storage
+google-cloud-storage>=1.31.0
```

Comment:
Can we add an upper pin to protect TNU from future breaking changes to google-cloud-storage?

Reply:
I just pinned 1.31.0, since it's the latest.